gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystem.java

```diff
@@ -63,6 +64,20 @@ public static CloudStorageFileSystem forBucket(String bucket, CloudStorageConfig
         new CloudStorageFileSystemProvider(), bucket, checkNotNull(config));
```

Comment:
not quite (because someone may have set default storage options), but I can return `forBucket(bucket, config, null)`. Done.